openhtf/core/test_record.py

```diff
@@ -72,7 +72,7 @@ class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['name', 'codeinfo'],
         {'measurements': None,
-         'start_time_millis': None, 'end_time_millis': None,
+         'start_time_millis': int, 'end_time_millis': None,
```

Comment:
https://docs.python.org/2/library/functions.html#int (it says `int(x=0)`)

Are you saying a platform we may support in the future may use non-epoch times? Or that someone may attempt to use `TestRecord` without our framework and throw non-epoch times in there? Because we call `time.time()` for all our timestamps.